pipeline/luigi/CompileVCFFiles.py

```diff
@@ -1499,7 +1524,8 @@ def requires(self):
         yield CopyG1KOutputToOutputDir(self.date, self.resources_dir, self.output_dir, self.file_parent_dir)
         yield CopyEXACOutputToOutputDir(self.date, self.resources_dir, self.output_dir, self.file_parent_dir)
         yield CopyEXLOVDOutputToOutputDir(self.date, self.resources_dir, self.output_dir, self.file_parent_dir)
-        yield CopySharedLOVDOutputToOutputDir(self.date, self.resources_dir, self.output_dir, self.file_parent_dir)
+        yield CopySharedLOVDOutputToOutputDir(date=self.date, resources_dir=self.resources_dir, output_dir=self.output_dir, file_parent_dir=self.file_parent_dir)
```

Comment:
any reason the syntax is different here? i.e. `date=self.date` instead of just `self.date`?

if it's more robust, could you make a follow up PR that makes the same change to all of the other calls as well?

Reply:
Mainly for robustness reasons. I'll keep that in mind if I come around refactoring this file (e.g. splitting it up etc). But before doing that, I'd like to have a closer look at the testing infrastructure in place

Reply:
makes sense. there's currently no testing infrastructure built around compileVCFfiles.py